doc/nrf/known_issues.rst

```diff
@@ -193,7 +197,7 @@ SLM logging over RTT is not available.
 Other issues
```

Comment:
As part of zephyr upmerge, CONFIG_LOG_PRINTK was change from default 'n' to 'y'. This may cause issue in applications which have used printk(). Together with a bug (CIA-892) when logging strings of more than 1023 characters, this may introduce new crashes/asserts.
We are still debating the resolution to CIA-892 for the NCS 2.3.0 release so not sure if this should be a known issue but I'll record it here so we don't forget to mention it.
Eventually, https://github.com/zephyrproject-rtos/zephyr/pull/54901 will resolve this.

Reply:
It seems there's been a mitigating hack for CIA-892 for asset tracker in PR https://github.com/nrfconnect/sdk-nrf/pull/10458 but it seems the real fix is not taken for 2.3.0. If that's the case, we need a statement either in known issues or probably preferably in release notes that `CONFIG_LOG_PRINTK` has been changed from 'n' to 'y'. Below is some example of what we could write:

"When logging a string with %s qualifier, the maximum length of any inserted string is 1023 bytes, or otherwise an assert or a crash will occur. Default value of `CONFIG_LOG_PRINTK` Kconfig option has been changed from 'n' to 'y' in this release so even use of printk() function may cause this issue unless the application disables this option."